content/en/docs/Reference/build-from-scratch.md

```diff
@@ -179,4 +179,30 @@ docker run -v --net host \
 {{% /tab %}}
 {{< /tabpane >}}
 
-This example is available in the `examples/byoi/fedora` directory of the [Kairos repository](https://github.com/kairos-io/kairos/tree/master/examples/byoi/fedora), where you can run `build.sh` to reproduce it.
\ No newline at end of file
+This example is available in the `examples/byoi/fedora` directory of the [Kairos repository](https://github.com/kairos-io/kairos/tree/master/examples/byoi/fedora), where you can run `build.sh` to reproduce it.
+
+## FIPSs compliant flavors
+
+To build a [FIPS](https://www.techtarget.com/whatis/definition/FIPS-Federal-Information-Processing-Standards) compliant version of Kairos, there are 2 requirements:
+
+- Your base image should be FIPS compliant
+- The kairos package from the fips category should be used (search for "fips" here: https://packages.kairos.io/)
```

Comment:
I'd add a notice mentioning that FIPS compliant kairos is compiled using Go 19 and the specific sha for the boringcrypto since FIPS complianc might mean different things depending on which certification you intend to have

Reply:
My previous comment probably belongs more in the last section you added about the binaries

Reply:
done with a new commit: https://github.com/kairos-io/kairos-docs/pull/40/commits/b5aecff9d4a93752396bbf65970107d605171277